library/threading.c

```diff
@@ -35,12 +36,17 @@ static void threading_mutex_init_pthread( mbedtls_threading_mutex_t *mutex )
     if( mutex == NULL )
         return;
 
-    mutex->is_valid = pthread_mutex_init( &mutex->mutex, NULL ) == 0;
+    mutex->is_valid = (pthread_mutex_init( &mutex->mutex, NULL ) == 0) ? 1 : 0;
```

Comment:
style: add space after first `(` and space before last `)`

Reply:
Fixing.